src/examples/java/com/nylas/examples/other/CalendarsExample.java

```diff
@@ -24,18 +24,26 @@ public static void main(String[] args) throws Exception {
 		NylasAccount account = client.account(conf.get("access.token"));
 		Calendars calendars = account.calendars();
 		String calendarEmail = null;
+		String accountId = null;
+		String calendarId = null;
 		for (Calendar calendar : calendars.list()) {
 			log.info(calendar);
 			if (calendar.getName().contains("@")) {
```

Comment:
is this checking if the name is an email? what about adding getEmail() to the calendar object and checking valid in there?

Reply:
Sorry, I'm not sure I'm following you here. Email isn't a field in a calendar object I don't think, and having a `getEmail` might confuse users into thinking email might be a field.

If I am understanding you correctly though, maybe having a utility method in the Calendar class called `nameIsEmail()` that checks if the name is indeed an email and returns a boolean value? but I'm not sure how often something like this would be used. Open to hearing your thoughts!